Documentation/configuration/metrics.rst

```diff
@@ -74,9 +74,9 @@ Events external to Cilium
 
 Cilium as a Kubernetes pod
 ==========================
-The Cilium `Prometheus reference configuration <https://github.com/cilium/cilium/blob/master/examples/kubernetes/addons/prometheus/prometheus.yaml>`_
-configures jobs that automatically collect pod metrics marked with the
-appropriate two labels.
+The Cilium prometheus reference configuration configures jobs that automatically
```

Comment:
... confguration _that_ configures jobs ...

Reply:
@techcet 

"The Cilium prometheus reference configuration that configures jobs that automatically" sounds grammatically incorrect. I think keeping it as is is OK.